polkadot/runtime/westend/src/weights/pallet_proxy.rs

```diff
@@ -66,6 +66,20 @@ impl<T: frame_system::Config> pallet_proxy::WeightInfo for WeightInfo<T> {
 	}
 	/// Storage: Proxy Proxies (r:1 w:0)
 	/// Proof: Proxy Proxies (max_values: None, max_size: Some(1241), added: 3716, mode: MaxEncodedLen)
+	/// The range of component `p` is `[1, 31]`.
+	fn proxy_propagate_error(p: u32, ) -> Weight {
```

Comment:
You also need to write a benchmark for this. This could also just be a copy of the `proxy` benchmark.